### ISSUE TITLE
Opacity values with 2 decimal points reported as error by Squiz.CSS.Opacity.SpacingAfterPoint

### ISSUE THREAD

**Issue by h3xx** [squizlabs/PHP_CodeSniffer#871]
Opacity values with 2 digits after the decimal point are detected as invalid by the Squiz Opacity sniff. They shouldn't be.

Rationale:

An opacity delta of `0.05` is actually a visible change, and useful for design:

![screenshot-2016-0120-0923](https://cloud.githubusercontent.com/assets/615684/12453211/d714459e-bf57-11e5-8d05-aa121b04dfc8.png)

**Possible feature:** There also isn't any way to tune this sniff to allow opacity values of arbitrary precision.

Steps to reproduce:

1. Create a CSS file with the following contents:

    #ghost {
        opacity: 0.25;
    }

2. Run `phpcs -s --standard=Squiz --sniffs=Squiz.CSS.Opacity ghost.css`.

This outputs:

    FILE: /home/dchurch/ghost.css
    ----------------------------------------------------------------------
    FOUND 1 ERROR AFFECTING 1 LINE
    ----------------------------------------------------------------------
     2 | ERROR | Opacity values must have a single value after the decimal
       |       | point (Squiz.CSS.Opacity.SpacingAfterPoint)
     ----------------------------------------------------------------------

Sniffs affected:
* Squiz.CSS.Opacity.SpacingAfterPoint

Files affected:
* CodeSniffer/Standards/Squiz/Sniffs/CSS/OpacitySniff.php

**Comment by gsherwood** [comment]
I think the bit you are missing here is that this sniff is part of the Squiz coding standard, and the graphic designers who decided on this part of standard wanted to ensure single digits were used, because that is what they use for Squiz products.

Like all coding standards, it's not for everyone. If you don't want this error in your custom standard, you can easily mute the error in your ruleset.xml file, either by using:
```xml
<rule ref="Squiz.CSS.Opacity.SpacingAfterPoint">
  <severity>0</severity>
</rule>
```
or by excluding it in an existing excludes list you may have:
```xml
<exclude name="Squiz.CSS.Opacity.SpacingAfterPoint"/>
```
Does that work for you?

**Comment by h3xx** [comment]
Why the name, though? `SpacingAfterPoint` doesn't communicate to me what's being picked up. Something like `DecimalPrecision` might be more accurate.

**Comment by gsherwood** [comment]
> Why the name, though?

It was probably a mistake I made when originally writing the sniff. I'll change it, but you'll need to change your ruleset after the next release to match the new code.

**Comment by gsherwood** [comment]
I've renamed the error code. Does this, and my previous comments, solve your issue?

**Comment by h3xx** [comment]
Yes-- at least partially.

Would be nice to have it adjustable, since there isn't another sniff like it in any of the other standards, but I'll take what I can get.
